core/base/inc/TSystem.h

```diff
@@ -401,8 +401,8 @@ class TSystem : public TNamed {
    virtual FILE           *OpenPipe(const char *command, const char *mode);
    virtual int             ClosePipe(FILE *pipe);
    virtual TString         GetFromPipe(const char *command);
-   virtual void            Exit(int code, Bool_t mode = kTRUE);
-   virtual void            Abort(int code = 0);
+   virtual void            Exit [[ noreturn ]] (int code, Bool_t mode = kTRUE);
+   virtual void            Abort [[ noreturn ]] (int code = 0);
    virtual int             GetPid();
    virtual void            StackTrace();
```

Comment:
> Another way would be to add as a new line above.

Let's try to move those two to the end of the section and add a new line:

```suggestion
   virtual int             GetPid();
   virtual void            StackTrace();
   
   [[ noreturn ]] virtual void Exit(int code, Bool_t mode = kTRUE);
   [[ noreturn ]] virtual void Abort(int code = 0);
```